### PR TITLE
Update customer-success.md

### PR DESCRIPTION
**Pull request by kumar-mayadata**

### CODE REVIEW

File: job-roles/customer-success.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ As a customer success engineer, you will be responsible for users and customers
 
 ### Customer Success Engineer
 
+* Expert knowledge on Linux OS(Centos, Ubuntu, Debian, RedHat) with at least 1-2 years experience in system administration and support role.
 * Customer Success Engineers have gained experience in installing the OpenEBS product and connecting them to the Director and in using Litmus.
 * Customer Success Engineers have the ability to determine the components that are potentially causing an issue.
 * Customer Success Engineers are comfortable editing and updating the OpenEBS, Litmus, and Director documentation based on the interactions with the customers.
```
Comment (kmova) [inline job-roles/customer-success.md:43]: n_: update this to OpenEBS and Kubera?

File: job-roles/customer-success.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ As a customer success engineer, you will be responsible for users and customers
 
 ### Customer Success Engineer
 
+* Expert knowledge on Linux OS(Centos, Ubuntu, Debian, RedHat) with at least 1-2 years experience in system administration and support role.
 * Customer Success Engineers have gained experience in installing the OpenEBS product and connecting them to the Director and in using Litmus.
```
Comment (kmova) [inline job-roles/customer-success.md:41]: n_: Probably change this to "experience in installing the OpenEBS and other products from MayaData"

File: job-roles/customer-success.md (hunk 1 above)
Comment (kumar-mayadata) [reply job-roles/customer-success.md]: Updated

File: job-roles/customer-success.md (hunk 2 above)
Comment (kumar-mayadata) [reply job-roles/customer-success.md]: updated

File: job-roles/customer-success.md [hunk 3]
```diff
@@ -31,21 +31,24 @@ As a customer success engineer, you will be responsible for users and customers
...
 * The individuals in this role are well versed in the concepts and have undergone the OpenEBS training and workshop.
-* The individuals in this role are expected to help customers by pointing in the right direction with the help of Documentation or KB articles, helping them navigate through OpenEBS, Director and Litmus.
+* The individuals in this role are expected to help customers by pointing in the right direction with the help of Documentation or KB articles, helping them navigate through OpenEBS and other products from MayaData.
 * The individuals are expected to become comfortable with tools like Slack and ZenDesk and help with ensuring the customer tickets are updated.
 * People are usually in this role for at least 3 months before becoming full-fledged Customer Success Engineers
 
 ### Customer Success Engineer
 
-* Customer Success Engineers have gained experience in installing the OpenEBS product and connecting them to the Director and in using Litmus.
+* Expert knowledge on Linux OS(Centos, Ubuntu, Debian, RedHat) with at least 1-2 years experience in system administration and support role.
```
Comment (bobek) [inline job-roles/customer-success.md:40]:
```suggestion
* Expert knowledge on Linux (Centos, Ubuntu, Debian, RedHat) with at least 1-2 years experience in system administration and support role.
```

**Review by bobek — APPROVED** [review]

**Review by kmova — APPROVED** [review]

**Pull request merged**